library/ssl_tls.c

```diff
@@ -5795,6 +5805,38 @@ static uint16_t ssl_tls12_preset_suiteb_sig_algs[] = {
 };
 #endif /* MBEDTLS_SSL_PROTO_TLS1_2 */
 
+
+/* As per rfc9151 */
+static const uint16_t ssl_preset_cnsa_sig_algs[] = {
+#if defined(MBEDTLS_KEY_EXCHANGE_ECDSA_CERT_REQ_ANY_ALLOWED_ENABLED) && \
```

Comment:
Should I do the rebase once code review is done? or before? Please suggest.

Reply:
I'm afraid that due to an increased workload, we've had to downgrade the priority of https://github.com/Mbed-TLS/mbedtls/issues/4602 and many other issues. So I can't promise a re-review any time soon. I'm sorry about this.

Given the time between reviews, it would be easiest for us if you rebase just before we're ready for re-review (we'll let you know). Preferably, `git range-diff origin/development OLD-COMMIT-ID NEW-COMMIT-ID` should give a usable result.